docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-api-request-for-quote-feature-integration.md

```diff
@@ -0,0 +1,647 @@
+
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker/quote-requests-rest-api:"^0.1.4" spryker/quote-request-agents-rest-api:"^0.1.4" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules have been installed:
+
+| MODULE                        | EXPECTED DIRECTORY <!--for public Demo Shops-->  |
+|-------------------------------|--------------------------------------------------|
+| QuoteRequestAgentsRestApi     | vendor/spryker/quote-request-agents-rest-api     |
+| QuoteRequestsRestApi          | vendor/spryker/quote-requests-rest-api           |
+| QuoteRequestsRestApiExtension | vendor/spryker/quote-requests-rest-api-extension |
+
+{% endinfo_block %}
+
+### 2) Set up the configuration
+
+Add following configuration to your project configuration:
+
+| CONFIGURATION                                     | SPECIFICATION                                                    | NAMESPACE                        |
+|---------------------------------------------------|------------------------------------------------------------------|----------------------------------|
+| CompanyUsersRestApiConfig::COMPANY_USER_RESOURCES | Specifies resources which are accessible only for company users. | Spryker\Glue\CompanyUsersRestApi |
+
+```php
+<?php
+
+namespace Pyz\Glue\CompanyUsersRestApi;
+
+use Spryker\Glue\CompanyUsersRestApi\CompanyUsersRestApiConfig as SprykerCompanyUsersRestApiConfig;
+use Spryker\Glue\QuoteRequestsRestApi\QuoteRequestsRestApiConfig;
+
+class CompanyUsersRestApiConfig extends SprykerCompanyUsersRestApiConfig
+{
+    /**
+     * @var array<string>
+     */
+    protected const COMPANY_USER_RESOURCES = [
+        QuoteRequestsRestApiConfig::RESOURCE_QUOTE_REQUESTS,
+        QuoteRequestsRestApiConfig::RESOURCE_QUOTE_REQUEST_CANCEL,
+        QuoteRequestsRestApiConfig::RESOURCE_QUOTE_REQUEST_REVISE,
+        QuoteRequestsRestApiConfig::RESOURCE_QUOTE_REQUEST_SEND_TO_CUSTOMER,
+    ];
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Check that specified resources are only accessible if authenticated customer has company user assigned.
+
+{% endinfo_block %}
+
+
+### 3) Set up transfer objects
+Generate transfers:
+
+```bash
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure the following transfers have been created:
+
+| TRANSFER                                | TYPE  | EVENT   | PATH                                                                          |
+|-----------------------------------------|-------|---------|-------------------------------------------------------------------------------|
+| RestAgentQuoteRequestsRequestAttributes | class | created | src/Generated/Shared/Transfer/RestAgentQuoteRequestsRequestAttributesTransfer |
```

Comment:
what about `RestAgentQuoteRequestsRequestAttributes::cartUuid` and `RestAgentQuoteRequestsRequestAttributes::metadata` properties?

Reply:
Since this is an IG for the whole feature, `RestAgentQuoteRequestsRequestAttributes` is a new transfer and it is specified in the list of new transfers.

Reply:
sorry, missed it